blueprints/fragment/files/app/__path__/__name__.js

```diff
@@ -1,6 +1,6 @@
-import DS from 'ember-data';
-import MF from 'ember-data-model-fragments';
+<%= importedModules.length ? `import { ${importedModules} } from '@ember-data/model';` : '' %>
+import Fragment from 'ember-data-model-fragments/fragment';
 
-export default MF.Fragment.extend({
+export default class <%= classifiedModuleName %> extends Fragment {
   <%= attrs %>
```

Comment:
Could we remove the space here because having it indented results in the first `attr` being indented too far?

```suggestion
<%= attrs %>
```

Reply:
@patocallaghan doing some tests with the generator, I've found that the result with attributes isn't valid. Look what happens when I type `ember g fragment foo bar:string`:

```
import { attr } from '@ember-data/model';
import Fragment from 'ember-data-model-fragments/fragment';

export default class Foo extends Fragment {
  bar: attr('string')
}
```

So I'm going to fix that too.